### PR TITLE
chore: update readmes

### PR DESCRIPTION
**Pull request by anthony9187**
## Description

update the readmes to reflect the updated project structure as well as include more relevant information

## Changes

- update root readme, remove solid package readme
- add theming.md
- add PR title check to the project workflows

## Testing
review the updated readmes

### CODE REVIEW

File: .github/workflows/pr-title-check.yml
Comment (cee-money) [inline .github/workflows/pr-title-check.yml:1]: nice!

File: .vscode/settings.json
Comment (cee-money) [inline .vscode/settings.json:1]: should these updates have been committed?

File: THEMING.md [hunk 1]
```diff
@@ -4,11 +4,161 @@
...
+        }
+      }
+    }
+  }
+}
+```
+
+## App Setup
+
+Themes are loaded at compile time by the app bundler (we recommend [vite](https://vitejs.dev/) which is built on top of [rollup](https://rollupjs.org/)). In your bundler config, you'll need add an import alias. We're working on a plugin to clean this up a little but for now:
```
Comment (ImCoolNowRight) [inline THEMING.md:161]: Should read "you'll need to add".

File: .vscode/settings.json
Comment (anthony9187) [reply .vscode/settings.json:1]: yep, unless anyone's against i have a couple of settings in my local settings.json that shouldnt be committed to the repo(theyre specific to my IDE setup)

Comment (cee-money) [reply .vscode/settings.json:1]: just for my benefit, any reason we chose these specific colors, and what's the benefit of adding these to contributors' workbenches - helping them differentiate between their various VS Code windows?

**Review by ImCoolNowRight — DISMISSED** [review]

**Review by ctoddy — DISMISSED** [review]

**Review by chiefcll — DISMISSED** [review]

File: .vscode/settings.json
Comment (anthony9187) [reply .vscode/settings.json:1]: oh my bad. i read @cee-money's as being about the gitignore commit not the settings commit. ill get that stuff removed

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,4 +0,0 @@
-{
-  "cSpell.words": ["autodocs", "lorum", "noto"],
-  "typescript.tsdk": "node_modules/typescript/lib"
-}
```
Comment (cee-money) [inline .vscode/settings.json:4]: it's probably fine to leave these properties if you wanted to @anthony9187 👍

**Review by cee-money — DISMISSED** [review]

**Review by cee-money — APPROVED** [review]
LGTM

**Pull request merged**